README.md

```diff
@@ -98,21 +99,22 @@ One can see all dependencies including Transitive Dependency by executing follow
 mvn dependency:tree
 ```
 
-###Exclude Dependencies
-If you wish to limit the number of run-time dependencies, and your project does not require the features named above, you can explicitly exclude them by adding exclusion tag.  
-***For Example:*** If you are not using *Always Encrypted Azure Key Vault feature* then you can exclude *azure-keyvault* dependency. Please see following snippet. 
+### Azure Key Vault and Azure Active Directory Authentication Dependencies
+Since the dependencies of ***Azure Key Vault*** and ***Azure Active Directory Authentication*** are optional, projects that require those 2 features need to redeclare the dependency explicitly in their pom files.
```

Comment:
Let's just say "projects that require either of the two features need to explicitly declare the dependency in their pom file".